### PR TITLE
Vertamedia Adapter  for 1.0 

### PR DESCRIPTION
**Pull request by Millerrok**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change

Update for vertamedia adapter to Prebid 1.0
Add vertamedia md file
Update tests for vertamedia adapter

- test parameters for validating bids
```
{
    bidder: 'vertamedia', 
            params: {
                aid: 332842
            }
}
```

### CODE REVIEW

File: modules/vertamediaBidAdapter.js [hunk 1]
```diff
@@ -1,119 +1,125 @@
...
+  /**
+   * Make a server request from the list of BidRequests
+   * @param bidRequests
+   * @param bidderRequest
+   */
+  buildRequests: function (bidRequests, bidderRequest) {
+    return bidRequests.map((bid) => {
+      return {
+        data: prepareRTBRequestParams(bid),
+        contentType: 'text/plain',
```
Comment (jaiminpanchal27) [inline modules/vertamediaBidAdapter.js]: `contentType` is `text/plain` by default and `withCrendentials` is also set to true. 
Both can be removed from here.

File: modules/vertamediaBidAdapter.js [hunk 2]
```diff
@@ -1,119 +1,125 @@
...
-});
+/**
+ * Configure new bid by response
+ * @param bidRequest {object}
+ * @param bidResponse {object}
+ * @returns {object}
+ */
+function createBid(bidResponse, bidRequest) {
+  return {
+    bidderCode: bidRequest.bidderCode,
```
Comment (jaiminpanchal27) [inline modules/vertamediaBidAdapter.js]: `bidderCode` will be set by bidderFactory. Can be removed from here.
Also no need to add `descriptionUrl`

File: modules/vertamediaBidAdapter.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+```
+ var adUnits = [{
+        code: 'div-test-div',
+        sizes: [[640, 480]], // ad size
+        bids: [{
+            bidder: 'vertamedia', // adapter name
+            params: {
+                aid: 332842
+            }
+        }]
```
Comment (jaiminpanchal27) [inline modules/vertamediaBidAdapter.md]: `}]` missing at the end.

**Comment by Gorokhov** [comment]
@jaiminpanchal27  thanx for review. I have updated the PR according to your comments. Please take a look.

Regarding descriptionUrl, it can be found at line 108. Please confirm that this is what you were looking for. 
Thanks in advance,

**Pull request merged**